lib/jsdom/living/node.js

```diff
@@ -191,3 +191,25 @@ module.exports.listOfElementsWithNamespaceAndLocalName = (namespace, localName,
     } });
   });
 };
+
+// true for all node types except document ones
+function isNode(node) {
+  return node && (
+    node.nodeType <= NODE_TYPE.COMMENT_NODE ||
+    node.nodeType === NODE_TYPE.DOCUMENT_FRAGMENT_NODE
+  );
+}
+
+// create a fragment (or just return a node for one item)
+exports.createNode = (document, nodes) => {
```

Comment:
This should be called convertNodesIntoNode and have a comment linking to https://dom.spec.whatwg.org/#converting-nodes-into-a-node . It should also have the parameter order switched, to match the spec.

It should also follow the algorithm there; this seems rather different.